static-catalog/implementations/DigitalGlobe/examples/DigitalGlobe_1B_product_pan.json

```diff
@@ -0,0 +1,151 @@
+{
+  "id": "103001004B316600_P002_MUL",
+  "type": "Feature",
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [
+            -105.160846396,
+            39.1656009422
+          ],
+          [
+            -104.947574355,
+            39.1750024943
+          ],
+          [
+            -104.949890373,
+            39.0512153242
+          ],
+          [
+            -105.161151214,
+            39.0436821016
+          ],
+          [
+            -105.160846396,
+            39.1656009422
+          ]
+        ]
+      ]
+    ]
+  },
+  "assets": [
+    {
+      "uri": "https://15NOV09180446-M1BS-056823192010_01_P002.TIF",
+      "metadata": {
+        "filetype": "geotif",
+        "content": "image",
+        "processing": {
+          "RadiometricallyCorrected": true
+        }
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.IMD",
+      "metadata": {
+        "filetype": "imd",
+        "content": "image metadata"
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.XML",
+      "metadata": {
+        "filetype": "xml",
+        "content": "image metadata"
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.TIL",
+      "metadata": {
+        "filetype": "til",
+        "content": "virtual raster image"
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.ATT",
+      "metadata": {
+        "filetype": "att",
+        "content": "attitude metadata"
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.EPH",
+      "metadata": {
+        "filetype": "eph",
+        "content": "ephemeral metadata"
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.GEO",
+      "metadata": {
+        "filetype": "geo",
+        "content": "geospatial metadata"
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.RPB",
+      "metadata": {
+        "filetype": "rpb",
+        "content": "rational polynomial metadata"
+      }
+    },
+    {
+      "uri": "https://15NOV09180446-P1BS-056823192010_01_P002.JPG",
+      "metadata": {
+        "filetype": "jpeg",
+        "content": "thumbnail"
+      }
+    }
+  ],
+  "properties": {
+    "bands": [
+      {
+        "common_name": "PAN",
+        "gsd": 0.512,
+        "center_wavelength": 627.0,
+        "effective_bandwidth": 361.0,
+        "image_band_index": 0
+      }
+    ],
+    "schema": "DigitalGlobeProduct",
+    "type": [
```

Comment:
Is this supposed to be feature_type ? The other two use that term. Or I'm missing some subtlety as to why this is different.

Reply:
Yes, I wasn't sure what we finally decided on for the name of the field.  Schema, type, product.....